pkg/api-server/testdata/resources/inspect/dataplanes/_rules/meshgateway.golden.json

```diff
@@ -12,6 +12,40 @@
  "rules": [
   {
    "fromRules": [],
+   "toResourceRules": [
+    {
+     "conf": [
+      {
+       "connectionTimeout": "2s",
+       "idleTimeout": "20s",
+       "http": {
+        "requestTimeout": "10s"
+       }
+      }
+     ],
+     "origin": [
+      {
+       "resourceMeta": {
+        "labels": {
+         "k8s.kuma.io/namespace": "kuma-demo",
+         "kuma.io/display-name": "mt-on-gateway"
+        },
+        "mesh": "default",
+        "name": "mt-on-gateway",
+        "type": "Mesh"
+       },
+       "ruleIndex": 0
```

Comment:
could you please add a test when `ruleIndex` is greater than 0?

Reply:
added